projects/file/Dockerfile

```diff
@@ -15,7 +15,10 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y make autoconf automake libtool shtool zlib1g-dev:i386
+RUN apt-get update
+RUN apt-get install -y make autoconf automake libtool shtool
+RUN apt-get install -y libbz2-dev      liblzma-dev      zlib1g-dev      libzstd-dev      liblz4-dev
+RUN apt-get install -y libbz2-dev:i386 liblzma-dev:i386 zlib1g-dev:i386 libzstd-dev:i386 liblz4-dev:i386 
```

Comment:
You should do update every time you do an install, otherwise caching can make your build fail randomly (your update will be out of date)